### PR TITLE
Add xontrib-fzf-widgets

### PR DESCRIPTION
**Pull request by shahinism**
I'm trying to port some of useful [fzf](https://github.com/junegunn/fzf) widgets developed for Bash/ZSH to Xonsh And thought that it might be a good idea to share it with the Xonsh community as a Xontrib.

### CODE REVIEW

**Comment by astronouth7303** [comment]
Can you add a news item?

Did we ever form an opinion about formally using `xip` in the xontribs listing?

**Comment by laerus** [comment]
No, but we can do that now?

**Comment by astronouth7303** [comment]
@shahinism Please change two things:

* Add a [news item](http://xon.sh/devguide.html#changelog)
* Please use `xip` instead of `pip` in the package installation directions (`xip` works better with homebrew and conda installations)

**Comment by laerus** [comment]
all your CI are belong to us.

File: news/add_fzf_widgets_extension.rst [hunk 1]
```diff
@@ -0,0 +1,14 @@
+**Added:** None
```
Comment (gforsyth) [inline news/add_fzf_widgets_extension.rst]: Hey @shahinism -- you need to remove the `None` from this line to resolve the test failures.  Thanks!

**Comment by laerus** [comment]
funny how tests fail about news item but news PR lint is OK

**Comment by laerus** [comment]
now the problem is that single backticks are not valid rst you need double backticks     ``

maybe we need a news formatter

**Comment by laerus** [comment]
bummer that seems like a bug with 3.6 and Travis, I think we can merge this.

**Pull request merged**